downloads/swan-lake-release-notes/swan-lake-preview4/RELEASE_NOTE.md

```diff
@@ -314,6 +315,10 @@ Executing a `fail` Statement will cause the control flow to transfer to the `on
     }
 ```
 
+##### Predeclared module prefixes
+
+The module prefix `t` (which is one of the `boolean`, `decimal`, `error`, `float`, `future`, `int`, `map`, `object`, `stream`, `string`, `table`, `typedesc` or `xml`) has been predeclared as referring to the `lang.t` lang library module now.
+
```

Comment:
Shall we add an example, along the lines of

Such modules can now be used without explicitly importing them.

```ballerina
public function main() {
    int max = 'int:max(1, 2, 3);
}
```

Reply:
Added.